test/e2e/page-objects/pages/network-manager.ts

```diff
@@ -66,40 +68,47 @@ class NetworkManager {
     await this.driver.clickElementSafe(this.networkListItem(chainId));
   }
 
-  // Method to select/click on a network item
-  async selectNetwork(networkName: string): Promise<void> {
-    console.log(`Selecting network: ${networkName}`);
-    await this.driver.delay(regularDelayMs);
-    await this.checkNetworkIsDeselected(networkName);
-    await this.driver.delay(regularDelayMs);
-    await this.driver.clickElementSafe(this.networkListItem(networkName));
-    await this.driver.delay(regularDelayMs);
-    await this.checkNetworkIsSelected(networkName);
+  async checkAllPopularNetworksIsSelected(): Promise<void> {
```

Comment:
Update NetworkManager page object. We cannot select multiple networks now and we do not render checkboxes.